www-client/dooble/dooble-1.56f.recipe

```diff
@@ -6,7 +6,7 @@ manager, and many others."
 HOMEPAGE="https://textbrowser.github.io/dooble/"
 COPYRIGHT="Dooble"
 LICENSE="BSD (3-clause)"
-REVISION="4"
+REVISION="5"
```

Comment:
Haven't checked a build, but why keep the older version if the newer version is fine?

Reply:
Meant to delete it....